docs/guides/expectations/how-to-create-and-edit-expectations-with-instant-feedback-from-a-sample-batch-of-data.md

```diff
@@ -2,4 +2,87 @@
 title: ✳ How to create and edit Expectations with instant feedback from a sample Batch of data
 ---
 
-This article is a stub.
+import Prerequisites from '../../guides/connecting_to_your_data/components/prerequisites.jsx'
+
+<Prerequisites>
+
+- Configured a [Data Context](../../tutorials/getting-started/initialize-a-data-context.md).
+
+</Prerequisites>
+
+Steps
+-----
+
+1. The ``--interactive`` mode denotes the fact that you are interacting with your data.  In
+    other words, you have access to a data source and can specify a Batch of data to be used to create Expectations
+    against.  ``--manual`` mode
+    (please see [How to create and edit Expectations based on domain knowledge, without inspecting data directly](./how-to-create-and-edit-expectations-based-on-domain-knowledge-without-inspecting-data-directly))
```

Comment:
Link is broken on Netlify

Reply:
Great catch -- checking...

Reply:
@cdkini It works locally!  Any ideas what to do? Reload/retry?..

Reply:
@cdkini Also seems to work here: `https://deploy-preview-3046--knoxpod.netlify.app/docs/guides/expectations/how-to-create-and-edit-expectations-based-on-domain-knowledge-without-inspecting-data-directly` -- please confirm.

Reply:
Hmm if it works and the build was successful, it's probably a rendering issue? Netlify might just take a second.

Let's give it a few minutes but I think there shouldn't be an issue @alexsherstinsky

Reply:
🙇